cms/templates/js/content-group-details.underscore

```diff
@@ -3,7 +3,7 @@
         <h3 class="title">
             <a href="#" class="toggle group-toggle <% if (showContentGroupUsages){ print('hide'); } else { print('show'); } %>-groups">
                 <i class="ui-toggle-expansion icon fa fa-caret-<% if (showContentGroupUsages){ print('down'); } else { print('right'); } %>"></i>
-                <%= name %>
+                <%- name %>
```

Comment:
I noticed this issue and decided to clean up the rest of the file at the same time.